build-docs.sh

```diff
@@ -3,5 +3,15 @@
 cd docs/
 
 rm -rf build/
-sphinx-build -b html . build -W
+
+cp -R pytket-docs-theming/_static .
+cp -R pytket-docs-theming/quantinuum-sphinx .
+cp pytket-docs-theming/conf.py .
+
+sphinx-build -b html . build -D html_title="pytket user guide"
```

Comment:
Can we keep the `-W` (which I assume treats warnings as errors)?

Reply:
added this back in after resolving some warnings in [6a6ab33](https://github.com/CQCL/pytket-docs/pull/389/commits/6a6ab33243d891f24ba6afea2e1936b23fbbc4f4) and [1691730](https://github.com/CQCL/pytket-docs/pull/389/commits/16917309e0fd05e29216a1c20c71a2561ef198f3).